tests/unicode_delimited_tuple_attribute_test.py

```diff
@@ -13,11 +15,15 @@
 class MyTuple(NamedTuple):
     country: str
     city: str
-    # should be Optional[int] but deserialization does not support it
-    zip_code: int = None  # type: ignore
+    zip_code: Optional[int] = None
```

Comment:
Whatever is happening here, do you want to address it separately?

Reply:
This is because of https://github.com/lyft/pynamodb-attributes/issues/32